README.md

```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
 
 To understand how to use this starter kit, follow the tutorial at [https://workadventu.re/map-building](https://workadventu.re/map-building).
 
+## Introduction
+
+- WorkAdventure is a open source virtual office solution
+- Offers spontaneous meetings without the hassle of setting a date
+- Maps can be designed and structured as pleased
+
+## Submitting a Map-Change
+
+- Open issue with suggetion for change
+- No one complains -> submit pull request with changes
+- Important! Concurrently working on the same map is very problematic
+  - Therefore, changes should be as small as possible and quickly transition from issue to PR
+
+## Working with Tiled
+
+Tiled: Intuitive tool for creating tilemaps
+
+### Tilesets
```

Comment:
Todo (separate commit?)

We should probably add some metadata, where we got the file from and how it is licensed.

Reply:
What do you mean by that?

Reply:
Something like this:          https://github.com/itemis-Nord/ces-workadventure-maps/blob/8e052257584937b6ba878e275587c829c4463f57/map.json#L1002

I think it makes sense to make it mandatory.